index.html

```diff
@@ -4,63 +4,146 @@
+            Awesome <span class="text-secondary">BOOKS</span>
+          </h1>
+          <ul class="book-list"></ul>
 
-      <form class="book-form">
-        <div class="form-group">
-          <input
-            type="text"
-            id="title"
-            name="title"
-            placeholder="title"
-            required
-          />
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th>Title</th>
+                <th>Author</th>
+                <th>ID</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody id="book-list"></tbody>
+          </table>
         </div>
-        <div class="form-group">
-          <input
-            type="text"
-            id="author"
-            name="author"
-            placeholder="author"
-            required
-          />
+      </section>
+
+      <section id="addnew" class="section-add-book">
+        <div class="container mt-4">
+          <h2>Add a new <span class="text-secondary">BOOK</span></h2>
+          <form class="book-form">
+            <div class="form-group">
+              <input
+                type="text"
+                id="title"
+                name="title"
+                placeholder="title"
+                required
+              />
+            </div>
+            <div class="form-group">
+              <input
+                type="text"
+                id="author"
+                name="author"
+                placeholder="author"
+                required
+              />
+            </div>
+            <button class="btn btn-primary btn-block" type="submit">
+              Add
+            </button>
+          </form>
+        </div>
+      </section>
+
+      <section id="contact" class="contact-section hide">
+        <div class="container mt-4">
+          <h2>Contact Information</h2>
+          <p>
+            Do have any question or you just want to say "hello" <br />
+            You can Reach out to us!
+          </p>
+          <ul>
+            <li>Our E-mail: books@awesome.com</li>
+            <li>Our Phone Number: +964-772-218-0404</li>
+            <li>Our Adress: Sulaymaniyah-Kurdistan-Iraq</li>
+          </ul>
         </div>
-        <button class="btn btn-primary btn-block" type="submit">
-          Add a new book
-        </button>
-      </form>
-    </div>
+      </section>
+      <!-- <section id="contact" class="contact-section hide">
+        <div class="row contact-container">
+          <div class="description">
+            <h1>Contact Information</h1>
+            <p>Do you have any question or you just want to say "hello"?</p>
+            <p>You can reach out to us through:</p>
+          </div>
+          <ul>
+            <li>info@awesomebooks.com</li>
+            <li>Tel: +964-772-218-0404</li>
+            <li>Sulaymaniyah-Kurdistan-Iraq</li>
+          </ul>
+        </div>
+      </section> -->
```

Comment:
Kindly remove the commented code. It makes your project looks messy, :+1: :smiley: :smiley:

Reply:
Good job following the advice of the previous reviewer